openid-4-verifiable-presentations-1_0.md

```diff
@@ -2,12 +2,12 @@
 title = "OpenID for Verifiable Presentations - Editor's draft"
 abbrev = "openid-4-vp"
 ipr = "none"
-workgroup = "connect"
+workgroup = "OpenID Connect"
```

Comment:
It wasn't right before but this isn't right either, is it? Should be dcp?

Reply:
That's as far as I made it or will make it in this PR this holiday weekend.

Reply:
OpenID4VP is an OpenID Connect working group specification.  The plan has always been for DCP to do a call for adoption for the specification after the next Implementer's Draft.

As a point of reference, when OpenID4VCI reached Implementer's Draft status after the formation of the DCP working group, [it was adopted by DCP](https://lists.openid.net/pipermail/openid-specs-digital-credentials-protocols/Week-of-Mon-20240415/000259.html).

The last [OpenID4VP Implementer's Draft](https://openid.net/specs/openid-4-verifiable-presentations-1_0-ID2.html) was before [the formation of the DCP working group](https://lists.openid.net/pipermail/openid-specs-digital-credentials-protocols/Week-of-Mon-20230904/000001.html), so this hasn't yet been done for OpenID4VP.

Reply:
Woah, really? I guess I'd just been assuming that transition had happened after all the hullabaloo around it whenever it happened. And not paying close enough attention. But you know what they say about assuming...

Reply:
To the extent that all the OIDF procedures around IPR, WGs, etc., actually matter, this seems rather problematic. But what do I know? I'm only on line 5 of one PR...

Reply:
If you can review the lines after line 5 as well, that would be appreciated.

Reply:
understood and acknowledging that doing so is on my todo list

Reply:
officially, it is AB/Connect Working Group. not going to die about the string value here, but just saying.